fpm-entrypoints/php-fpm-jit.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+   # Ensure the socket directory has correct permissions
+   mkdir -p /var/run
+   chown -R www-data:www-data /var/run
+   chmod 755 /var/run
+
+   # Start PHP-FPM
+   php-fpm --fpm-config /app/deploy/conf/php-fpm.conf -c /app/deploy/conf/php.ini
```

Comment:
php-fpm-jit.sh, php-fpm-no-session.sh, and php-fpm-opcache.sh are equal, right? Why would you need 3 files in this case?

Reply:
oh, sorry, I forgot to delete entrypoint files!
jit adds just minor few per cents, I dropped it as a separate test